SwrveSDK/SDK/Track/Swrve.h

```diff
@@ -2,6 +2,7 @@
 #import "SwrveInterfaceOrientation.h"
 #import "SwrveReceiptProvider.h"
 #import "SwrveResourceManager.h"
+#import "SwrveBaseCampaign.h"
```

Comment:
👍  cool, am just afraid it will get removed at later date accidentally. Actually, if message center sample fails compilation if this is removed then thats good enough for me!